app/Http/Controllers/RankingController.php

```diff
@@ -20,27 +20,53 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Beatmap;
+use App\Models\UserStatistics;
+use App\Transformers\UserStatisticsTransformer;
+use Request;
+
 class RankingController extends Controller
 {
-    protected $section = 'ranking';
+    protected $section = 'rankings';
 
-    public function getOverall()
-    {
-        return view('ranking.overall');
-    }
+    const PAGE_SIZE = 20;
+    const MAX_RESULTS = 10000;
 
-    public function getCountry()
+    public function index($mode, $type, $page = 1)
     {
-        return view('ranking.country');
-    }
+        $max_pages = ceil($this::MAX_RESULTS / $this::PAGE_SIZE);
```

Comment:
a bit weird seeing constant accessed using `$this` instead of `static`. Also, camelCase.